cmiles/tests/test_utils.py

```diff
@@ -162,33 +162,15 @@ def test_canonical_order_rd():
 @pytest.mark.parametrize('input, output', [('COC(C)c1c(Cl)ccc(F)c1Cl', False),
                                            ('C[C@@H](c1c(ccc(c1Cl)F)Cl)OC', False),
                                            ('O=O', True),
-                                           ('[CH3:1][CH2:3][CH2:4][CH3:2]', False)])
+                                           ('[CH3:1][CH2:3][CH2:4][CH3:2]', False),
```

Comment:
You are a master of unit tests and reproducibility, high on a mountaintop above us mere mortals.